app/DoctrineMigrations/Version20230920192807.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Application\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+/**
+ * Auto-generated Migration: Please modify to your needs!
+ */
+final class Version20230920192807 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return '';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('CREATE TABLE notification (id SERIAL NOT NULL, name VARCHAR(64) NOT NULL, enabled BOOLEAN DEFAULT \'true\' NOT NULL, PRIMARY KEY(id))');
```

Comment:
A problem with this implementation, is that it would need the table to be filled on a fresh CoopCycle instance. 
Migrations are not executed on a fresh instance, so it would need to be filled in the `SetupCommand`
IMHO it would be better to not rely on the data in the table.